.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,43 @@
+---
+name: Bug report
+about: Report a bug
+title: "[BUG] - "
+labels: bug
+assignees: ''
+
+---
+
+**Internal/External**
```

Comment:
It wasn't immediately obvious to me what to do with these - can we make them checkboxes insetead?

Reply:
Do we need this one in particular? I think it'll likely be obvious from the username. Maybe it's still helpful.

Reply:
I copied that from the node repository. But I agree we probably don't need it, this information is already on Github.

Reply:
Yeah, and with these things the longer we make it the more likely people just ignore it.